.rubocop_todo.yml

```diff
@@ -34,6 +34,8 @@ Metrics/BlockLength:
 # Configuration parameters: CountComments, CountAsOne.
 Metrics/ClassLength:
   Max: 2006
+  Exclude:
```

Comment:
Is there a reason for excluding the this test file in rubocop?

Reply:
Thank for your question,
I saw the CI test failed here https://travis-ci.com/github/sendgrid/sendgrid-ruby/jobs/471986228#L3744.
Because the lines increased when I add more tests to this file. I thought these tests failed because of the effect of this rule. And I had checked the git blame of this rule here https://github.com/sendgrid/sendgrid-ruby/blame/6da9f9eaa6167eb2a64aea4406e52691da5431ca/.rubocop_todo.yml.
However, I don't have any guidelines for the 2006 number. I think these file's lines still to increase not only in this change, and neither in the future. So I added it.

Reply:
Hi Jennifer,
Could you confirm about the answer for me?

Reply:
I agree with this reasoning!